portfolio/src/main/webapp/c.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Chinese</title>
+        <link rel="stylesheet" href="style.css">
+        <script src = "./js/index.js"> </script>
+    </head>
+
+    <body id = "content">
+        <!--nav var-->
+		<div id="navbar" class="navbar">
+		  <a href="./index.html">Home</a>
+		  
+		  <div class="cdropdown">
+		    <a class="cdropbtn" href="./c.html">Chinese
+		    </a>
+		    <div class="cdropdown-content">
+		      <a href="#c">Chinese</a>
+		      <a href="#photos">China Photos</a>
+		    </div>
+          </div>
+        </div>
+
+        <h1 id="c"></h1>
```

Comment:
How about naming it "china-container" or something like that. It is good to have names that make it clear what the css id/class name is. That way when you look at the css file, you don't need to refer back to the html file to figure out what "c" was.

Reply:
done